source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,44 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-01-26
+
+### Highlights
+
+#### Channel Sidebar General Availability
+ - The new channel sidebar is enabled system-wide by default. Mattermost now gives users flexibility to organize channels and direct messages into custom, collapsible sidebar categories. Users now gain full personalization of their sidebar to improve productivity, reduce clutter and focus on what matters. Features include custom, collapsible sidebar categories, unread channel management, and new channel menus.
```

Comment:
```suggestion
 - The new channel sidebar is enabled system-wide by default. Mattermost now gives users flexibility to organize channels and direct messages into custom, collapsible sidebar categories. Users now gain full personalization of their sidebar to improve productivity, reduce clutter, and focus on what matters most. Features include custom, collapsible sidebar categories, unread channel management, and new channel menus.
```

Reply:
@amyblais it was always possible to do this on cloud since the new sidebar has always been turned on for cloud users. I think we should focus instead on the new updates offered:

- configurable unreads section
- Sorting of categories by recent activity or alpha (or manually as available before)

cc// @cwarnermm if you want to help with word smithing ;)

Reply:
@amyblais I'm in the process of updating the product documentation and will soon have some verbiage I can offer for these release notes.